.circleci/config.yml

```diff
@@ -32,6 +33,9 @@ jobs:
       - run:
           name: Build
           command: yarn build
+      - slack/status:
+          fail_only: true
+          failure_message: ':red_circle: A $CIRCLE_JOB has failed! You should probably do something about that...'
```

Comment:
🤯  Mind blown. Cool trick

Reply:
We can do so much more with these orbs too! This was the simplest implementation that I did just for prototyping after I pushed a failed build.